app/controllers/candidate_interface/after_sign_in_controller.rb

```diff
@@ -13,7 +13,7 @@ def interstitial
         flash[:warning] = "You have already selected #{course_from_find.name_and_code}."
         redirect_to candidate_interface_course_choices_review_path
       elsif current_application.maximum_number_of_course_choices?
-        error_message_key = current_application.apply_1? ? 'errors.messages.too_many_course_choices' : 'errors.messages.apply_again_course_already_chosen'
+        error_message_key = 'errors.messages.too_many_course_choices'
```

Comment:
Minor: maybe move this directly into the `I18n.t` method below and skip the variable now?

Reply:
Good shout. Thanks - have updated ♻️